data/migrations/V0272__add_ofec_schedule_d_report.sql

```diff
@@ -0,0 +1,102 @@
+/*
+This migration file will create a materialized view for the schedule D endpoint with
+added report information
+issue #4758 (Add report-level information to the debts endpoint)
+*/
+
+-- ---------------
+-- ofec_sched_d_mv
+-- ---------------
+CREATE MATERIALIZED VIEW public.ofec_sched_d_mv AS
+    SELECT
+        sd.sub_id,
+        sd.cmte_id,
+        sd.image_num,
+        sd.filing_form,
+        sd.link_id,
+        sd.line_num,
+        sd.tran_id,
+        sd.file_num,
+        sd.orig_sub_id,
+        sd.cmte_nm,
+        sd.cred_dbtr_id,
+        sd.cred_dbtr_nm,
+        sd.cred_dbtr_l_nm,
+        sd.cred_dbtr_f_nm,
+        sd.cred_dbtr_m_nm,
+        sd.cred_dbtr_prefix,
+        sd.cred_dbtr_suffix,
+        sd.cred_dbtr_st1,
+        sd.cred_dbtr_st2,
+        sd.cred_dbtr_city,
+        sd.cred_dbtr_st,
+        sd.creditor_debtor_name_text,
+        sd.entity_tp,
+        sd.nature_debt_purpose,
+        sd.outstg_bal_bop,
+        sd.outstg_bal_cop,
+        sd.amt_incurred_per,
+        sd.pymt_per,
+        sd.cand_id,
+        sd.cand_nm,
+        sd.cand_nm_first,
+        sd.cand_nm_last,
+        sd.cand_office,
+        sd.cand_office_st,
+        sd.cand_office_st_desc,
+        sd.cand_office_district,
+        sd.conduit_cmte_id,
+        sd.conduit_cmte_nm,
+        sd.conduit_cmte_st1,
+        sd.conduit_cmte_st2,
+        sd.conduit_cmte_city,
+        sd.conduit_cmte_st,
+        sd.conduit_cmte_zip,
+        sd.action_cd,
+        sd.action_cd_desc,
+        sd.schedule_type,
+        sd.schedule_type_desc,
+        sd.election_cycle,
+        sd.pg_date,
+        sd.rpt_tp,
+        sd.rpt_yr,
+        public.date_or_null(fr.cvg_start_dt::integer) as coverage_start_date,
+        public.date_or_null(fr.cvg_end_dt::integer) as coverage_end_date
```

Comment:
I don't see this function being used for date conversion anywhere else. Mostly we use expression::type for date conversion. Both cvg_start_dt 
and cvg_end_dt are timestamp without time zone.